lib/bridge/MatrixHandler.js

```diff
@@ -960,7 +965,7 @@ MatrixHandler.prototype._onJoin = Promise.coroutine(function*(req, event, user)
                 try {
                     yield kickIntent.kick(
                         event.room_id, user.getId(),
-                    `Connection limit reached for ${room.server.domain}. Please try again later.`
+                    `IRC connection failure.`
```

Comment:
Not sure if we should change this: the error at least helps identify a problem to us.

Reply:
Well, the error is wrong in the case of banned users. I'm wondering if there is a sane way to pass it down the exception path.

Reply:
fair, we should just set up proper monitoring on the i-line usage instead.